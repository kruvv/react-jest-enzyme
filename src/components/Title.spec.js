import { shallow } from 'enzyme';
import React from 'react';
import Title from './Title';


describe('<Title />', () => {

    it('should correctly render component Title without props', () => {
        const component = shallow(<Title />);
        expect(component).toMatchSnapshot();
    });
    it('should correctly render component Title with props', () => {
        const component = shallow(<Title title="Test title" />);
        expect(component).toMatchSnapshot();
    });
});
