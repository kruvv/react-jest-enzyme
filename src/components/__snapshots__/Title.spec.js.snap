// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> should correctly render component Title with props 1`] = `
<h1
  className="title"
>
  Test title
</h1>
`;

exports[`<Title /> should correctly render component Title without props 1`] = `
<h1
  className="title"
>
  Simple title
</h1>
`;
